util/messages.po

```diff
@@ -51,7 +51,7 @@ msgstr ""
 #: mod/profile_photo.php:198 mod/wall_attach.php:80 mod/wall_attach.php:83
 #: mod/item.php:166 mod/uimport.php:15 mod/cal.php:306 mod/regmod.php:108
 #: mod/editpost.php:19 mod/fsuggest.php:80 mod/allfriends.php:23
-#: mod/contacts.php:387 mod/events.php:195 mod/follow.php:54 mod/follow.php:118
+#: mod/contact.php:387 mod/events.php:195 mod/follow.php:54 mod/follow.php:118
```

Comment:
This file will need to be completely regenerated because the file `mod/contact.php` doesn't exist anymore after your PR. As such, you didn't need to change it, but it's okay.